backend/app/utilities/emailHelper.js

```diff
@@ -0,0 +1,72 @@
+const nodemailer = require('nodemailer')
+const mustache = require('mustache')
+const path = require('path')
+const fs = require('fs')
+const { PROJECT_DIR } = require('../constants')
+
+
+async function renderTemplateAsEmail(templatePath, emailVars) {
+  const templateDir = path.join(PROJECT_DIR, 'app/templates')
+  const fullTemplatePath = path.join(templateDir, templatePath)
+  const templateData = await fs.readFileSync(fullTemplatePath)
```

Comment:
do you need an `await` here? I thought readFileSync was synchronous

Reply:
changed it